tutorials/plot_background_filtering.py

```diff
@@ -483,15 +524,232 @@ def plot_signal(x, offset):
 plt.show()
 
 ###############################################################################
-# Filtering in MNE-Python
-# =======================
+# Some pitfalls of filtering
+# ==========================
+# Multiple recent papers have noted potential risks of drawing
+# errant inferences due to misapplication of filters.
+#
+# Low-pass problems
+# -----------------
+# Filters in general, especially those that are acausal (zero-phase), can make
+# activity appear to occur earlier or later than it truly did. As
+# mentioned in VanRullen 2011 [3]_, investigations of commonly (at the time)
+# used low-pass filters created artifacts. However, such deleterious effects
```

Comment:
How do you mean? I think this is correct:

https://1931-1301584-gh.circle-artifacts.com/0//home/ubuntu/mne-python/doc/_build/html/auto_tutorials/plot_background_filtering.html#references

The ones that appear multiple times look different from the others but I
think that's how numpydoc intends to do it.